hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/handler/RSProcedureHandler.java

```diff
@@ -48,7 +48,7 @@ public void process() {
     try {
       callable.call();
     } catch (Throwable t) {
-      LOG.error("Error when call RSProcedureCallable: ", t);
+      LOG.error("pid=" + this.procId, t);
```

Comment:
This log is too little?

Reply:
I changed it so that it logs the pid same way we do it every where else. The thread/class/level already provides what we add here. I just removed redundancy.